### PR TITLE
Removed shadow from filter chips

### LINKED ISSUE #135: filter chips have a shadow
### Steps to Reproduce

1. Go to 'https://compass-test.tuerantuer.org/?'
2. See Filter area on the left side

### Additional Information

<details>
  <summary>Traceback</summary>

  ```
  ```
</details>

### PR DESCRIPTION
**Pull request by amikable-tech**
### Short description

Removed shadow from filter chips

### Proposed changes

- Modified the filter chips to show shadow only on mouse hover.

### Side effects

None

### Resolved issues

Fixes: #135

### CODE REVIEW

**Review by JoeyStk — CHANGES_REQUESTED** [review]
Thanks! This already looks good. Could you please also change the shadow in[ this file and line]( https://github.com/digitalfabrik/integreat-compass/blob/develop/integreat_compass/cms/templates/offers/offer_form.html#L76)?

File: integreat_compass/cms/templates/offers/offer_form.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h2>{% translate "Offer details" %}</h2>
                                 <div>
                                     {{ choice.tag }}
                                     <label for="{{ choice.id_for_label }}"
-                                           class="px-4 py-2 text-lg bg-white rounded-full text-center shadow-lg peer-checked:bg-primary peer-checked:text-gray-700 peer-checked:border-none">
+                                           class="px-4 py-2 text-lg bg-white rounded-full text-center hover:shadow-lg peer-checked:bg-primary peer-checked:text-gray-700 peer-checked:border-none">
```
Comment (JoeyStk) [inline integreat_compass/cms/templates/offers/offer_form.html]:
```suggestion
                                           class="px-4 py-2 text-lg bg-tertiary font-normal rounded-full text-center hover:shadow-lg peer-checked:bg-primary peer-checked:text-gray-700 peer-checked:border-none">
```

Sorry, that I didn't think of this before. But without the shadow it's difficult to see the chips/tags now. Therefore I would suggest to adapt the look of the chips from "all offers"

**Review by JoeyStk — APPROVED** [review]
Thank you! I'm happy with it now, but maybe it makes sense to have a second person have a look at it :)

File: integreat_compass/cms/templates/offers/offer_form.html (hunk 1 above)
Comment (charludo) [reply integreat_compass/cms/templates/offers/offer_form.html]: Maybe a custom class is already warranted? I think it's the same design in 3 places now, right?

**Pull request merged**